website/docs/components/card/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,49 @@
+## Usage
+
+### When to use
+
+Use a **Card Container** to help design more consistent cards.
+
+### Types
+
+#### Static Cards
```

Comment:
I noticed we use capital C here and a few lines down (L13) we use 'Interactive card'
```suggestion
#### Static cards
```